### DISCUSSION TITLE
Exclude subdirectory as part of dbt_project configuration?

### DISCUSSION THREAD

**Discussion by p-hebert** [dbt-labs/dbt-core#5466] [Q&A] [answered]
Hi!

I would like to be able to colocate my data science jupyter notebooks that allow me to generate my seeds with the seeds themselves. The issue I am facing is that one of said data science uses a csv file as source data to create the seed, and said csv file is picked up automatically by `dbt seed`, no matter what I seem to put in dbt_project.yml.

Here's the file structure:

```
dbt/
  seeds/
    _datascience/
      my_seed_src_csv.csv # I want to exclude this one
      my_seed.ipynb
    my_seed.csv
    my_seed.yaml
  ...
  dbt_project.yml
```

Here's the `seeds` clause of the dbt_project.yml file:

```
...
seed-paths: ["seeds"]

...
seeds:
  +schema: "{{env_var('DATASET_PREFIX')}}_common_raw"
  # resource-paths
  +seeds._datascience:
    enabled: false
```

I tried `seeds._datascience`, `_datascience`, `+seeds._datascience`, `+_datascience`.

Am I missing something in my configuration or this is something that is not supported by dbt?

Thanks a lot!

**Comment by joellabes** [comment]
Hey @philippefutureboy, the most obvious thing that jumps out is that your directory is called `_workbooks`, not `_datascience`. Is that a typo in the post?

**Reply by p-hebert** [reply]
Hi @joellabes ! Indeed that's a typo in the post 🙃 
EDIT: Fixed!

**Comment by jaypeedevlin** [comment, marked as answer]
Try adding the project name into your seeds config:

```yml
seeds:
  +schema: "{{env_var('DATASET_PREFIX')}}_common_raw"
  project_name:
    # resource-paths
    _datascience:
      +enabled: false
```

**Reply by p-hebert** [reply]
It works!
Thanks @jaypeedevlin :)
